modules/react/loading-dots/stories/examples/Accessible.tsx

```diff
@@ -1,6 +1,39 @@
 import React from 'react';
 import {LoadingDots} from '@workday/canvas-kit-react/loading-dots';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {SecondaryButton} from '@workday/canvas-kit-react/button';
+import {createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';
```

Comment:
`createStyles` and  `createStencil` need to be outside a render function. Otherwise the performance will be suboptimal.

```suggestion
import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';

const loadingStyles = createStyles({
  backgroundColor: base.licorice300,
  padding: system.space.x3,
});
```